src/main/java/seedu/duke/flashutils/types/FlashCardSet.java

```diff
@@ -62,16 +61,39 @@ public void viewFlashCards(String moduleName) {
     }
 
     public void performFlashBang() {
-        int i = 0;
+        int num = 0;
+        int correctAnswers = 0;
+        int wrongAnswers = 0;
+
         for (Card card : flashCardSet) {
-            Ui.printResponse("Flashcard no." + i + "\n\t" + card.getQuestion());
+            Ui.printResponse("Flashcard no." + num + "\n\t" + card.getQuestion());
             Ui.printResponse("Reveal the answer? (y/n)");
-            String ans = Ui.getRequest();
-            if (ans.equals("y")) {
+            String revealAnswer = Ui.getRequest();
+            if (revealAnswer.equals("y")) {
                 Ui.printResponse("Answer:\n\t" + card.getAnswer());
             }
-            i++;
+
+            // For tracking of correct/wrong answers
+            Ui.printResponse("Did you get the correct answer? (y/n)");
+            String answerCorrect = Ui.getRequest();
+            if (answerCorrect.equals("y")) {
+                correctAnswers++;
+
+            } else if (answerCorrect.equals("n")) {
+                wrongAnswers++;
+
+            } else {
+                Ui.printResponse("Please enter 'y' or 'n'.");
```

Comment:
If the code hits this else statement it should probably go back to the previous iteration so that it more accurately tracks in case users hit wrong inputs